### PR TITLE
Get id only from window.location.href

### PR DESCRIPTION
**Pull request by Cloud11PL**

### CODE REVIEW

**Review by jwm0 — COMMENTED** [review]
Looks good but the test needs to be adjusted

**Comment by Cloud11PL** [comment]
> Looks good but the test needs to be adjusted

@jwm0 Fixed

**Review by jwm0 — APPROVED** [review]

File: test/index.test.ts [hunk 1]
```diff
@@ -6,11 +6,19 @@ Object.defineProperty(window.document, "referrer", {
...
+  },
+  writable: true,
+});
+
 // eslint-disable-next-line
 import { actions, DispatchResponseEvent, createApp } from "../src";
 
 describe("createApp", () => {
-  const domain = "http://test-domain.com/?id=appid";
+  const domain = "http://test-domain.com/";
```
Comment (magul) [inline test/index.test.ts:21]: I know that nobody recall any usage of `createApp` with provided `domain`, but before @Cloud11PL changes there was just a domain here, and now we're providing url with schema na host, which in that test make it incompatible with `createApp` execution without `domain` provided.

File: test/index.test.ts [hunk 2]
```diff
@@ -6,11 +6,19 @@ Object.defineProperty(window.document, "referrer", {
   value: origin,
   writable: true,
 });
+
+Object.defineProperty(window, "location", {
+  value: {
+    href: origin + "?id=appid",
```
Comment (magul) [inline test/index.test.ts:12]: we could even provide it with a domain (meaning
```
href: origin + "?domain=saleor.domain&id=appid",
```

Comment (magul) [reply test/index.test.ts:12]: I will also allow us to test `createApp` without `domain` provided (which we don't have covered right now and actually is only recognised way used in wild).

**Review by magul — APPROVED** [review]

**Pull request merged**